models/articles.models.js

```diff
@@ -1,7 +1,9 @@
 const db = require('../db/connection')
 
 exports.fetchArticle = (article_id) => {
-    return db.query(`SELECT * FROM articles WHERE article_id = $1`, [article_id]).then(({ rows }) => {
+
+    return db.query(`SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.body, articles.created_at, articles.votes, COUNT(comments.article_id) AS comment_count, articles.article_img_url FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id WHERE articles.article_id = $1 GROUP BY articles.article_id`, [article_id])
```

Comment:
If you want to shorten this query back down you can do `SELECT articles.*, ` then your comment count :)